main.go

```diff
@@ -144,3 +148,16 @@ func main() {
 		panic(err)
 	}
 }
+
+func loadInput(dumpFile string) *os.File {
+	if dumpFile != "" {
+		fmt.Printf("\nloading dump file from path: %s\n", dumpFile)
```

Comment:
nit; Suggest uppercasing first letter i.e., "\nLoading ..." since this will print on a new line

Reply:
making this as resolved